tools-and-examples/auto_update_exaop/auto_update_exaop.py

```diff
@@ -110,34 +130,50 @@ def prepare_update(cluster, cluster_path, storage):
     logger.info('%s: EXAStorage is offline now', func_name)
 
     #stop cluster service on all nodes
-    for node_name in cluster.getNodeList():
-        node = ssl_server_proxy(cluster_path + '/' + node_name)
-        node.stopClusterServices()
-        timeout = time.time() + LS_TIMEOUT
-        while node.getNodeState()['status'] != 'Suspended':
-            if time.time() > timeout:
-                logger.error('%s: Timeout while suspending node %s', func_name, node_name)
-                return 1
+    if str(cluster.getEXAoperationMaster()) not in cluster.getNodeList():
+        for node_name in cluster.getNodeList():
```

Comment:
In theory, it could change in between the calls. However it would be very unlikely. I think the node running Exaoperation would need to change from the license server to another data node at that exact moment.